Index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jordan| HOME</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@1,300&family=Poppins:wght@400;600;700&family=Raleway:wght@300&family=Roboto:ital,wght@1,300&display=swap"
+        rel="stylesheet">
```

Comment:
you can import this google file into your common or main css file, so this way you won't need to add it again and again into all your html file.

Reply:
yeah made another css file main.css imported there